File: app/src/main/java/com/neome/api/ent/agent/msg/MsgAgentActiveStatusUpdate.java
// neome.ai API - do not change
//

package com.neome.api.ent.agent.msg;

import com.neome.api.meta.base.Types.EntId;
import com.neome.api.nucleus.base.msg.Msg;

@SuppressWarnings("unused")
public class MsgAgentActiveStatusUpdate extends Msg
{
  public boolean active;

  public EntId entId;
}

File: app/src/main/java/com/neome/api/ent/agent/msg/MsgAgentActiveStatusUpdate.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.ent.agent.msg

import com.neome.api.meta.base.Types.EntId
import com.neome.api.nucleus.base.msg.Msg

interface MsgAgentActiveStatusUpdate : Msg
{
  val active: Boolean
  val entId: EntId
}